vizportal/payload.py

```diff
@@ -1,19 +1,20 @@
-from typing import Dict, List, Optional
+from typing import Dict, List, Optional, Union
 from vizportal.req_options import FilterClauseBuilder, SortBuilder
 
+
 class PayloadBuilder:
     """Builds a payload for a request to the Vizportal API.
 
     ...
 
     Attributes
     ----------
-    _payload : Dict
+    _payload : Dict[str, Union[str, Dict[str, Union[int, Dict[str, int]], List[str]]]]
         Protected internal instance of the payload.
 
     Properties
     ----------
-    payload : Dict
+    payload : Dict[str, Union[str, Dict[str, Union[int, Dict[str, int]], List[str]]]]
```

Comment:
I think for some of these really nested ones, it might be beneficial to annotate it as:
Dict[str, Any]